obal/__init__.py

```diff
@@ -27,6 +24,8 @@
 
 
 def find_packages(inventory_path):
+    from ansible.inventory.manager import InventoryManager
```

Comment:
You could even move these inside the if statement.

Reply:
true